src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationService.kt

```diff
@@ -793,6 +793,71 @@ class AdjudicationService(
     }.let { CreateHearingResultAwardResponses(it) }
   }
 
+  @Audit
+  fun updateCreateAndDeleteHearingResultAwards(adjudicationNumber: Long, chargeSequence: Int, requests: UpdateHearingResultAwardRequest): UpdateHearingResultAwardResponses {
+    val party = adjudicationIncidentPartyRepository.findByAdjudicationNumber(adjudicationNumber)
```

Comment:
just a naive cut and paste of the update code to get this going.
TODO - updates and deletes